src/sql/workbench/contrib/editData/browser/gridParentComponent.ts

```diff
@@ -499,15 +501,19 @@ export abstract class GridParentComponent extends Disposable {
 	 */
 	protected abstract tryHandleKeyEvent(e: StandardKeyboardEvent): boolean;
 
-	resizeGrids(): void {
+	resizeGrid(): void {
 		const self = this;
 		setTimeout(() => {
-			for (let grid of self.renderedDataSets) {
-				grid.resized.fire();
-			}
+			setTimeout(() => self.onResize(), 100);
```

Comment:
is there a need to have `setTimeout` call recursively call `setTimeout` vs. calling `onResize()` directly?

Reply:
going to call it directly